src/tests/multihost/alltests/conftest.py

```diff
@@ -738,6 +781,44 @@ def remove_nslcd():
     request.addfinalizer(remove_nslcd)
 
 
+@pytest.fixture(scope='class')
+def misc_test(session_multihost, request):
```

Comment:
This fixture is specific to 1 test only. Can we move it to the test file? It will help to have a clear name to the fixture and update the fixture scope.

Reply:
Done